### ISSUE TITLE
[IX] Selection rectangles or Click To Fill color boxes not rendered in the right place

### ISSUE THREAD

**Issue by gabriel-piles** [huridocs/uwazidocs#7393]
When using the 'click to fill' feature in a PDF text while editing it in the Library, the colored rectangles that indicate editable fields are displayed correctly on top of the text. However, when opening the same entity in the IX UI, the colored rectangles are not positioned correctly above the text. Please refer to the following images.

![Screenshot From 2024-10-21 14-58-45](https://github.com/user-attachments/assets/d44f6eee-ca3a-46f1-b8de-baa172ff1425)
![Screenshot From 2024-10-21 15-03-43](https://github.com/user-attachments/assets/4405b9c8-9e7f-4f50-9187-706aa9a24f66)

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by txau** [comment]
@aphilop @konzz we seem to have a new problem related to the size of the PDF on the side panel, so we may need to promote this issue to high priority.

Now the document is rendered oversized:

![image](https://github.com/user-attachments/assets/81183795-0053-498c-acad-1c0da86b7782)

Tested on FF and Chrome

**Comment by aphilop** [comment]
@txau @konzz 
This is fine for me. Changing it to priority high and adding it to the backlog.

**Comment by txau** [comment]
For this particular one it would be great to also understand how we broke a functionality that was working and why our tests didn't detect it.

**Comment by serena-huridocs** [comment]
Hi everyone! I am writing to double-check with you if the reported issue was fully resolved by https://github.com/huridocs/uwazi/pull/7456. @hyebin-bina and I have noticed some issues with Click to fill and Selection rectangles while testing. 
<img width="1434" alt="Screenshot 2024-12-02 at 10 44 20" src="https://github.com/user-attachments/assets/32e89ffd-dda8-4e2c-be3d-1d2171a31e9d">

**Comment by Zasa-san** [comment]
> Hi everyone! I am writing to double-check with you if the reported issue was fully resolved by #7456. @hyebin-bina and I have noticed some issues with Click to fill and Selection rectangles while testing. 

Hi Serena! This is fixed, but some selections made during the period the bug existed will remain slightly off. These selections need to be redone, unfortunately. This issue should not be present on any new selections made on the sidepanel or de main document view. If new selections still show problems, please reopen the issue with what you find.

**Comment by serena-huridocs** [comment]
Thank you so much @Zasa-san! Bina and I tested this last week and noticed issues with selecting fragments of the text (the highlighted areas would go way beyond the areas we targeted). I also tested it yesterday before reporting the issue and noticed that the issue persisted with inconsistency between the selected fragment of the text and the text displayed in the Click to Fill field (on the attached screenshot **Agenda item 3** -> **uties in accordance with those**). However, upon checking today, it seems that everything has been resolved. 

Sorry for any inconvenience!
